sim-cli/src/main.rs

```diff
@@ -28,6 +28,10 @@ struct Cli {
     expected_pmt_amt: Option<u64>,
     #[clap(long, short)]
     capacity_multiplier: Option<f64>,
+    // DEV-ONLY args, set hide = true so they are not show in help
```

Comment:
Wonder whether this should be dev-only? Some folks in user interviews said they wouldn't really need the result file (would go straight to the node to see what happened), so I can see it being useful beyond dev. 

Weak opinion loosely held.

Reply:
I don't have a strong opinion here tbh, I found it useful for my testing setup so I figured it may be for other devs too, but if this is a legit use-case we can simply advertise it.

Reply:
Let's surface it for now? We're not drowning in flags (yet 😅 )